util/validators/jsonSchema/rest.ts

```diff
@@ -0,0 +1,207 @@
+import { Housing, Kita, Post, Refugee } from 'util/interfaces/rest';
+import { JSONSchemaType } from 'ajv';
+
+export const NewHousingREST: JSONSchemaType<Housing> = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Housing',
+  description: 'Data for a new Housing',
+  type: 'object',
+  properties: {
+    housing: {
+      type: 'object',
+      properties: {
+        housing_type: {
+          type: 'string',
+          minLength: 1,
+        },
+        people_limit: {
+          type: 'integer',
+          minimum: 1,
+        },
+        people_assigned: {
+          type: 'integer',
+          minimum: 0,
+          nullable: true,
+        },
+        size: {
+          type: 'number',
+          minimum: 1,
+          maximum: 300,
+        },
+        shared_bathroom: {
+          type: 'boolean',
+          nullable: true,
+        },
+        rooms: {
+          type: 'number',
+          minimum: 1,
+          maximum: 15,
+          nullable: true,
+        },
+        rent: {
+          type: 'number',
+          minimum: 0,
+          maximum: 3000,
+        },
+        info: {
+          type: 'string',
+          minLength: 0,
+          nullable: true,
+        },
+        citizen_id: {
+          type: 'number',
+          minimum: 1,
+        },
+      },
+      required: ['housing_type', 'size', 'rent', 'people_limit', 'citizen_id'],
```

Comment:
Hier ist citizen_id nicht optional, in der Spezifikation ist es das aber. Das einmal einheitlich festlegen, egal in welche Richtung